src/components/StrongPasswordInput/styled/input.js

```diff
@@ -0,0 +1,111 @@
+import styled from 'styled-components';
+import Input from '../../Input/styled/input';
+import attachThemeAttrs from '../../../styles/helpers/attachThemeAttrs';
+import { COLOR_WARNING } from '../../../styles/colors';
+
+const hasLeftIcon = props => props.icon && props.iconPosition === 'left';
+const hasRightIcon = props => props.icon && props.iconPosition === 'right';
+
+const WARNING_SHADOW = `0 0 2px ${COLOR_WARNING}`;
+
+const StyledInput = attachThemeAttrs(styled(Input))`
```

Comment:
You cannot use COLOR_WARNING. Can you do

`
const StyledInput = attachThemeAttrs(styled(Input)).attrs(props => {

const shadowWarning = `0 0 2px ${props.palette.warning.main}`;
return { shadowWarning };
})
`